agent/Makefile

```
@@ -3,15 +3,32 @@ include $(REPO_HACK_DIR)/tools.mk
-.PHONY: build test tidy lint fmt
+.PHONY: build test test-cuda-helper tidy lint fmt
 
 build:
 	GOOS=linux GOARCH=amd64 go build -o bin/agent ./cmd/agent
 	GOOS=linux GOARCH=amd64 go build -o bin/nsrestore ./cmd/nsrestore
 
-test:
+test: test-cuda-helper
 	go test ./...
 
+test-cuda-helper:
```

checkmake (0.3.2)

[warning] Target body for "test-cuda-helper" exceeds allowed length of 5 lines (16).

(maxbodylength)